src/modules/image/index.ts

```diff
@@ -353,7 +353,7 @@ export class ImageModule {
    * @param options Options for generating a data uri.
    * @param options.width The width of the image. Defaults to `640`.
    * @param options.height The height of the image. Defaults to `480`.
-   * @param options.color The color of the image. Defaults to `grey`.
+   * @param options.color The color of the image. Defaults to a random color.
```

Comment:
Both the `@param` description and the field description should be the same/similar.

```suggestion
   * @param options.color The color of the image. Must be a color supported by svg. Defaults to a random color.
```

Reply:
Looks like you forgot this one. Or did you mark this as resolved intentionally?

Reply:
Oh! So sorry I totally missed this one

Reply:
The color of the image. Must be a color supported by svg. it should be this right? same as description

Reply:
Plus the `Defaults to ...` part because that's not present in the `@param` block.